Platform/Teams/Triage/Rules of Engagement with Triage.md

```diff
@@ -0,0 +1,35 @@
+# Preface
+This document will help all team members working on VA.gov understand when it is appropriate to engage the Triage team for support and provide relevant information to assist in routing reported issues to other appropriate teams.  Because of the limited number of resources on Triage, they can not be a "catch-all" for every issue reported on the platform and every application.  Please read the following document carefully in order to determine the appropriate path for you incident.
```

Comment:
```suggestion
This document will help all team members working on VA.gov understand when it is appropriate to engage the Triage team for support and provide relevant information to assist in routing reported issues to other appropriate teams.  Because of the limited number of resources on Triage, they can not be a "catch-all" for every issue reported on the platform and every application.  Please read the following document carefully in order to determine the appropriate path for your incident.
```